starter-code/index.html

```diff
@@ -1,31 +1,64 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-  <title>Instagram</title>
-</head>
-<body>
-  Instagram
+  <head>
+    <title>Instagram</title>
+    <link rel="stylesheet" href="./css/main.css" />
+  </head>
 
-  Capture and Share the World's Moments
+  <body>
+    <section>
+      <!--IMAGE IPHONE-->
+      <div class="imageIphone">
+        <img src="./images/phones.png" alt="images of iphone" />
+      </div>
 
-  Instagram is a fast, beautiful and fun way to share your life with friends and family.
+      <!--RIGHT CONTAINER-->
+      <div class="right-container">
+        <!--HEADER-->
+        <header class="header">
+          <div><img src="./images/brand.png" alt="" /></div>
+          <button>
+            <img src="./images/home.png" alt="" /> <a href="#0">Log in</a>
+          </button>
+        </header>
 
-  Take a picture or video, choose a filter to transform its look and feel, then post to Instagram &mdash; it's that easy. You can even share to Facebook, Twitter, Tumblr and more. It's a new way to see the world.
+        <!--TITLE & PARAGRAPH-->
+        <div class="text">
+          <div class="text-paragraph">
+            <h1>
+              Capture and Share <br />
+              the World's Moments
+            </h1>
+            <p>
+              Instagram is a <b>fast, beautiful </b>and <b>fun </b>way to share
+              your life with friends and family.
+            </p>
+            <p>
+              Take a picture or video, choose a filter to transform its look and
+              feel, then post to Instagram &mdash; it's that easy. You can even
+              share to Facebook, Twitter, Tumblr and more. It's a new way to see
+              the world.
+            </p>
+            <p>Oh yeah, did we mention it's free?</p>
+          </div>
+          <!--ANDROID & GOOGLE-->
+          <div class="link">
+            <a href="#0"><img src="./images/badge-iphone.png" alt=""/></a>
+            <a class="b" href="#0">
+              <img src="./images/badge-android.png" alt=""
+            /></a>
+          </div>
+        </div>
+      </div>
+    </section>
 
-  Oh yeah, did we mention it's free?
-
-  Download on the App Store
-  Get it on Google Play
-
-  About Us
-  Support
-  Blog
-  Press
-  API
-  Jobs
-  Privacy
-  Terms
-
-  &copy; 2014 Instagram
-</body>
+    <!--FOOTER-->
+    <footer>
```

Comment:
As this is a kind of list - as a menu for ex - you could put all the lists inside a < ul > list :)
```
<footer>
   <ul>
      <li><a href="#0">About Us</a></li>
      <li><a href="#0">Support</a></li>
      <li><a href="#0">Blog</a></li>
      <li><a href="#0">Press</a></li>
      <li><a href="#0">API</a></li>
      <li><a href="#0">Jobs</a></li>
      <li><a href="#0">Privacy</a></li>
      <li><a href="#0">Terms</a></li>
      <li><p>&copy; 2014 Instagram</p></li>
   </ul>
</footer>
```